def test_addition():
    assert 1 + 1 == 2

def test_incorrect_addition():
    assert 1 + 1 == 3  # <- намеренная ошибка
